ensemble/volren/volume_bounding_box.py

```diff
@@ -9,6 +10,9 @@ class VolumeBoundingBox(ABCVolumeSceneMember):
     Volume.
     """
 
+    # The number of points should be 8 after the bounding box is computed
+    number_of_points = Int(0)
```

Comment:
This is trait is ad-hoc for testing. You need to expose what you create in the public API, instead.

For example, I think it would make sense to have a trait containing the `outline`. The test would check that its number of points as `outline.output.number_of_points`